scipy/stats/_continuous_distns.py

```diff
@@ -4171,6 +4171,121 @@ def _entropy(self, c):
 wrapcauchy = wrapcauchy_gen(a=0.0, b=2*pi, name='wrapcauchy')
 
 
+class gennorm_gen(rv_continuous):
+    """A generalized normal continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `gennorm` is [1]::
+
```

Comment:
ReST syntax: you need a trailing underscore for the reference, like so: [1]_ 
Or just omit it here (but keep it below).